generate-release/src/release_notes.rs

```diff
@@ -37,40 +37,35 @@ pub fn generate_release_notes(
         // 64 is completely arbitrary but felt long enough and is a nice power of 2
         file_name.truncate(64);
 
-        // TODO should probably add some weight for sorting
-        notes_metadata.push(format!(
-            r#"[[release_notes]]
-file_name = "{file_name}.md"
-"#,
+        // get the list of contributors to this PR
+        let mut contributors = HashSet::new();
+        let author = format!("@{}", pr.user.login);
+
+        if let Ok(pr_contributors) = get_contributors(client, &commit, &pr) {
+            for c in pr_contributors {
+                contributors.insert(c);
+            }
+        }
+
+        contributors.remove(&author);
+
+        let mut authors = vec![author];
```

Comment:
Is this to always have the PR creator in the first position? If that's the case I'd suggest adding a comment

Reply:
Yup, that's the idea.